### ISSUE TITLE
Proposal Feedback

### ISSUE THREAD

**Issue by gatoWololo** [upenn-cis198-2018f/final-project-substrate#2]
Oooh, I really like this project. I know just the basics about cryptocurrencies and blockchains, and had no idea about substrate. So I'm excited to see how it works out for you guys!

**Comment by zixuanzh** [comment]
@gatoWololo we did a little bit of deep dive into Substrate today. It provides core components of any blockchains, consensus, networking, client nodes. Developers can then build custom logic module on top of it. We are planning to build a well-tested bridge module. This involves working with their relatively complex API. Substrate is also very general purpose so we are working with generics and traits a lot. 

Question tho, i dont think we can fork into this repo. Should we just copy the code from substrate in our initial commit with credit and then build on top of it?

Their repo is at paritytech/substrate. Thanks!

**Comment by gatoWololo** [comment]
Yeah, you can just clone their repo into your local final-project repo, and push it.

Do you need to develop on top of the substrate repo? Or is it just a crate you include in your project. Whichever way, do what needs to be done!

**Comment by zixuanzh** [comment]
Just pushed it to our repo, we will be building a custom component within the substrate repo. Thanks!
